core/backend/src/CloudSqlite.ts

```diff
@@ -675,7 +720,7 @@ export namespace CloudSqlite {
       /** The name of the database within the container. */
       dbName: string;
     }) {
-      this._container = createCloudContainer({ ...args.props, writeable: true });
+      this._container = createCloudContainer({ writeable: true, ...args.props });
```

Comment:
Motivation for switching order of `writeable` and `...args.props`? The new way, if someone passes you an object with a `writeable` property, it will overwrite your `true`.

Reply:
Yes, that's what i wanted. If you don't supply it, the default is `true`. But you can now open a "readonly" container using the `DbAccess` class by passing `writeable: false` (though all writeable methods will fail.)